sceptre/cli/drift.py

```diff
@@ -68,9 +68,12 @@ def drift_detect(ctx: Context, path: str):
 
 @drift_group.command(name="show", short_help="Shows stack drift on running stacks.")
 @click.argument("path")
+@click.option(
+    "-D", "--drifted", is_flag=True, default=False, help="Filter out in sync resources."
```

Comment:
Why capital D here? Why not just `-d`?

Reply:
I would have expected `-d` to have been taken by `--debug` although I see we are not using short options at all. Perhaps it should just be `--drifted` with no short option?